src/main/kotlin/com/example/noswimmingserver/domain/book/domain/BookLike.kt

```diff
@@ -0,0 +1,21 @@
+package com.example.noswimmingserver.domain.book.domain
+
+import com.example.noswimmingserver.domain.common_user.domain.User
+import javax.persistence.*
```

Comment:
🚫 **[ktlint]** <no-wildcard-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wildcard import (cannot be auto-corrected)